app/jobs/run_submissions_job.rb

```diff
@@ -0,0 +1,28 @@
+class RunSubmissionsJob < ApplicationJob
+  queue_as :submission
+
+  def perform(*submission_ids)
+    submissions = Submission.find(submission_ids)
+    submissions.each do |submission|
+      run_submission submission
+    end
+  end
+
+  def run_submission(submission)
+    tests = Test.where(problem_id: submission.problem_id)
+
+    tests.each do |test|
+      execution = Execution.find_or_initialize_by(submission_id: submission.id, test_id: test.id)
```

Comment:
If we're going to make Execution a relationship between Submission and Test, we need to remember not to allow updates of Tests, or Executions could be based on stale test data.